security/openssl-tests.py

```diff
@@ -31,13 +31,20 @@ def setUp(self):
         Install the basic packages to support openssl
         '''
         # Check for basic utilities
+        run_type = self.params.get('type', default='upstream')
```

Comment:
any reason to change move this line up ?

Reply:
I can use this variable to check for additional packages to be installed. Those additional packages are required for upstream only case.